pkg/remote/aws/apigatewayv2_mapping_enumerator.go

```diff
@@ -38,6 +39,15 @@ func (e *ApiGatewayV2MappingEnumerator) Enumerate() ([]*resource.Resource, error
 			return nil, remoteerror.NewResourceListingError(err, string(e.SupportedType()))
 		}
 		for _, mapping := range mappings {
+			if mapping.Stage == nil || mapping.ApiId == nil {
```

Comment:
Are those attributes only used for displaying better human-readable attributes or are they also used in a middleware logic somewhere ? 

My point is that we should avoid skipping resources if we are only unable to retrieve "optional" attributes

Reply:
Used only for human-readable attributes

Reply:
Thanks @wbeuil super cool to have your API gateway knowledge here 🙏🏻

If it's not mandatory I think we should do a fix to ignore those attributes when we don't have them, but we should avoid to skip the resource, that will lead to inconsistency and thus false positive drifts